frontend/nextjs_frontend/orassistant-frontend/.gitignore

```diff
@@ -0,0 +1,36 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
any reason these src folders are nested in another folder `orassistant-frontend`? if not, prefer it to be under `nextjs_frontend` directly